quantized_mesh_tile/editable_terrain.py

```diff
@@ -0,0 +1,487 @@
+# -*- coding: utf-8 -*-
+import numpy
+import os
+
+import numpy as np
+from future.utils import old_div
+
+from quantized_mesh_tile import TerrainTile
+from quantized_mesh_tile.llh_ecef import LLH2ECEF
+from quantized_mesh_tile.terrain import MAX, lerp
+from quantized_mesh_tile.utils import triangleArea
+from . import cartesian3d as c3d
+
+null_normal = [0, 0, 0]
+np_null_normal = np.array([0, 0, 0])
+MIN = 0
+
+
+class EditableTerrainTile(TerrainTile):
+    """
+    Helper class to edit a given terrain tile on the edges.
+    Changes are possible on specified Heights, Triangles and Normals.
+
+
+    """
+
+    def __init__(self, *args, **kwargs):
+        super(EditableTerrainTile, self).__init__(*args, **kwargs)
+        self.is_index_dirty = False
+        self._changed_heights = []
+        self._file_path = None
+        self._gzipped = False
+        self._triangles = []
+        self.name = None
+
+    def set_name(self, name):
+        self.name = name
+
+    def get_edge_vertices(self, edge):
```

Comment:
This library uses `camelCase`, even if now I would probably go with the `snake_case` can we keep it this way?

Reply:
getEdgeIndices